admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/service/persistence/PersistenceManagerImpl.java

```diff
@@ -422,8 +429,16 @@ protected void postFetchValidation(PersistencePackage persistencePackage, Persis
                         .filter(sectionCrumb -> sectionCrumb.getOriginalSectionIdentifier().equals(foreignKey.getManyToField()))
                         .findFirst();
                     if (property != null && sectionCrumbOptional.isPresent()
-                        && !property.getValue().equals(sectionCrumbOptional.get().getSectionId())) {
-                        throw new SecurityServiceException("Post fetch validation: Access denied");
+                            && !property.getValue().equals(sectionCrumbOptional.get().getSectionId())) {
+                        ExtensionResultHolder<String> extensionResultHolder = new ExtensionResultHolder<String>();
+                        if (extensionManager.getProxy().checkParentId(property.getValue(),foreignKey.getForeignKeyClass(), extensionResultHolder) != ExtensionResultStatusType.NOT_HANDLED ){
+                           String parentID = extensionResultHolder.getResult();
```

Comment:
AbstractParentIdServiceExtensionHandler.checkParentId() always return ExtensionResultStatusType.NOT_HANDLED;

Is it correct?